threedworld-mit/tdw
# Fixed: rotate object commands translate incorrectly

Closes #218

# How to test:

1. Download and run the test build
  - [Linux](https://www.dropbox.com/s/v06rmkpd1bf7lo8/TDW_Linux_rotate_centroid.tar.gz?dl=0)
  - [OS X](https://www.dropbox.com/s/jrb48qqa63x8ii7/TDW_OSX_rotate_centroid.tar.gz?dl=0)
2. Run the controller posted in #218 

Result: Object rotates as expected.

# Changelog

### Build

- Fixed: `rotate_object_by`, `rotate_object_to`, and `robot_object_to_euler_angles` incorrectly translate the object if `use_centroid == True`.
